src/odemis/gui/cont/acquisition/cryo_acq.py

```diff
@@ -63,7 +64,22 @@
 
 # tmp flag for odemis advanced mode
 # TODO: remove and replace once the licenced version is released
-ODEMIS_ADVANCED_FLAG: bool = False
+def get_license_enabled() -> bool:
+    """
+    Temporary function to get the license status.
+    Allows to enable/disable without changing the code.
+    """
+    enabled  = False
+    odemis_advanced_config = os.path.abspath(os.path.join(conf.file.CONF_PATH, "odemis_advanced.config"))
+    if os.path.exists(odemis_advanced_config):
+        config = configparser.ConfigParser(interpolation=None)
+        config.read(odemis_advanced_config)
+        enabled = config["licence"].get("enabled", "False") == "True"
+ 
```

Comment:
Could you remove the space here, to make the linter happy again.

Reply:
done